backend/src/main/java/com/hackner/musicompass/model/Artist.java

```diff
@@ -5,18 +5,36 @@
 import lombok.Data;
 import lombok.NoArgsConstructor;
 import org.springframework.data.annotation.Id;
+
+import java.util.Date;
 import java.util.List;
+import java.util.Objects;
 
 @Data
 @Builder
 @AllArgsConstructor
 @NoArgsConstructor
 public class Artist {
 
+
     @Id
     private String artistName;
+    private Date saveDate;
```

Comment:
If you want to exclude a single field from the equals & hashcode you can do it like this instead of overriding the methods manually:
```java
@EqualsAndHashCode.Exclude
private Date saveDate
```

Reply:
thx. much more elegant !!!